docs/api-docs/management-apis/email-templates/email-template-examples.md

```diff
@@ -0,0 +1,156 @@
+# Email Templates Code Examples
+
+<div class="otp" id="no-index">
+
+### On this page
+
+- [Updating text](#updating-text)
+- [Adding a button](#adding-a-button)
+- [Changing logo size](#changing-logo-size)
+- [Changing font size](#changing-font-size)
+
+</div>
+
+This article provides examples of how to customize email templates. The examples are for use in the the recently updated templates. You can enable the new version of transactional emails by going to **Marketing> Transactional Emails** and clicking **Try New Experience**. When you are ready to use a template, make sure you **Enable** the template from the respective dropdown. 
+
+Code examples below will demonstrate how to update text, add a button, change logo size, and change the font size.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+NOTE: It is important to preview all changes and style your emails to be accessible for everyone.  
+</div>
+</div>
+</div> 
+
+## Updating text
```

Comment:
Is the information in this section replicated in the KB? 

If so, I recommend linking to it instead of recreating it here since any change to the instructions will need to be updated in two separate places by two separate teams down the line.